api/models/ssl.go

```diff
@@ -31,7 +31,7 @@ type SSL struct {
 
 type SSLs []SSL
 
-func CreateSSL(app, process string, port int, body, key string, chain string, secure bool) (*SSL, error) {
+func CreateSSL(app, process string, port int, arn string, body, key string, chain string, secure bool) (*SSL, error) {
```

Comment:
Every arg does have a type. If an arg name doesn't have one immediately following it, the closest type to the right of it is its type.

It's some sugar so you can do (foo, bar, baz string) instead of (foo string, bar string, baz string).

So the idiomatic thing here would be to drop the ` string` after arn and key, since they're all the same type as chain.